src/asserts/bank-identifier-code-assert.js

```diff
@@ -9,7 +9,7 @@ import { Validator, Violation } from 'validator.js';
  * Bic regex.
  */
 
-const bic = /^[a-zA-Z]{6}[a-zA-Z0-9]{2}([a-zA-Z0-9]{3})?$/;
+const bic = /^[A-Z]{6,6}[A-Z2-9][A-NP-Z0-9]([A-Z0-9]{3,3}){0,1}$/i;
```

Comment:
Isn't `{6,6}` redundant? It's equivalent to `{6}` AFAIK. Same for `{3,3}`.

Reply:
IMO I'd rather leave it as it's defined on the XSD.

Reply:
Fair enough.